src/components/SearchBarPopup/SearchBarPopup.tsx

```diff
@@ -0,0 +1,159 @@
+        newResults[elementsIndex] = { ...newResults[elementsIndex], isAdded: !newResults[elementsIndex].isAdded}
+        this.setState({
+            results: newResults,
+            resultsOpen: true
+        })
+    }
+
+    componentDidMount(): void {
+        document.addEventListener("mousedown", this.handleClickOutside, false);
+    }
+
+    componentWillUnmount(): void {
+        document.removeEventListener("mousedown", this.handleClickOutside, false);
+    }
+
+    private handleClickOutside = (event: MouseEvent) => {
+        if (this.container.current && event.target instanceof Node) {
+            if (this.container.current.contains(event.target)) {
+                return;
+            }  else {
+                // TODO: backend, send the updated result list to the backend
+            }
+        }
+    };
+    
+    private renderSearchResults() {
+        const { results } = this.state;
+        // console.log(this.state.resultsOpen)
+        if (this.state.resultsOpen && _.isArray(results) && !_.isEmpty(results)) {
+            return (
+                <div className="results-container">
+                    <ul className="results-list">
+                        {results.map((result) => (
+                            // temporarily using result as key. Change to some id later
+                            // TODO: replace with valid href
+                            // TODO:<a href> remount the entire component, fix when connecting to backend.
+                            <li className="results-entry" key={result["name"]} onClick={() =>this.addRemovePlace(result["id"])}>
+                                <a href="placeholder">
+                                    <span className="placeName">{result["name"]}</span>
+                                    <span className="address">{result["address"]}</span>
+                                </a>
+                                <a href="placeholder">                                    
+                                <FontAwesomeIcon className={result["isAdded"] ? "checked" : "checked icon-hidden"} icon="check" fixedWidth /></a>
+                            </li>
+                        ))}
+                    </ul>
+                </div>
+            );
+        }
+    }
+
+    render(){
+        return(
+            <div className="searchBarPopup">
+                <div className="contents">
+                    <h1 className="title"> Find and add places </h1>
+                    <div className="search-bar-wrapper">
+                        <form className="search-bar">
+                            <input
+                                type="search"
+                                placeholder="Search for..."
+                                className="search-input"
+                                onChange={this.handleOnSearchInputChange.bind(this)} 
+                            />
+                            <button type="submit" className="search-button">
```

Comment:
Sorry, design change. I was thinking... it might make more sense to remove this search button as we're not redirecting them to any new page. What do you guys think?

Reply:
Yea, I was going to ask you about this, but forgot. 
I think this search button should be removed as well.